tracer/test/test-applications/integrations/LogsInjection.Log4Net/LogsInjection.Log4Net.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <ApiVersion Condition="'$(ApiVersion)' == '' AND $(TargetFramework.StartsWith('net4')) ">1.2.11</ApiVersion>
-    <ApiVersion Condition="'$(ApiVersion)' == '' AND !$(TargetFramework.StartsWith('net4')) ">2.0.12</ApiVersion>
+    <ApiVersion Condition="'$(ApiVersion)' == '' AND !$(TargetFramework.StartsWith('net4')) ">3.0.0</ApiVersion>
```

Comment:
I think this means we aren't testing v2 anymore, but it _seems_ that we don't have any changes (maybe) with v3, so unsure if we want to test v2 still or not.

Reply:
Still testing v2, just defaulting to v3